### PR TITLE
Refactor/inspection

### PR DESCRIPTION
**Pull request by mathieupost**
Fixed a lot of inspection messages from IntelliJ!

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/menubar/LastOpenedController.java [hunk 1]
```diff
@@ -42,24 +42,23 @@ public LastOpenedController(Application application) {
 	@Override
 	public void actionPerformed(ActionEvent e) {
 		String chosenfile = e.getActionCommand();
-		File file = new File(chosenfile);
+		File nodeFile = new File(chosenfile);
```
Comment (kwendel) [inline src/main/java/tudelft/ti2806/pl3/menubar/LastOpenedController.java]: this file can be a folder, nwk tree or node/edge file. So naming it nodeFile is not logic.

**Comment by kwendel** [comment]
Code looks good!

**Comment by Delfil** [comment]
Ok

File: src/main/java/tudelft/ti2806/pl3/menubar/LastOpenedController.java (hunk 1 above)
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/menubar/LastOpenedController.java]: Ah ok, will fix.

**Comment by Delfil** [comment]
See comment on commit. Except for that, all good.

**Pull request merged**